website/docs/d/nsxt_network_dhcp_binding.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_network_dhcp_binding"
+sidebar_current: "docs-vcd-datasource-nsxt-network-dhcp-binding"
+description: |-
+  Provides a data source to read NSX-T Org VDC network DHCP bindings.
+---
+
+# vcd\_nsxt\_network\_dhcp\_binding
+
+Provides a data source to read NSX-T Org VDC network DHCP bindings.
+
+-> This data source requires VCD 10.3.1+
+
+## Example Usage
+
+```hcl
+data "vcd_nsxt_network_dhcp" "pools" {
+  org = "cloud"
+
+  org_network_id = data.vcd_network_routed_v2.parent.id
+}
+data "vcd_nsxt_network_dhcp_binding" "binding1" {
+  org            = "cloud"
+  org_network_id = data.vcd_nsxt_network_dhcp.pools.id
+  name           = "Binding-one"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization. Optional if defined at provider level
+* `org_network_id` - (Required) The ID of an Org VDC network. **Note**  (`.id` field) of
+  `vcd_network_isolated_v2`, `vcd_network_routed_v2` and `vcd_nsxt_network_dhcp` suite here
```

Comment:
The parenthesis look strange :) Maybe like this (same suggestion for the resource doc below):

```suggestion
* `org_network_id` - (Required) The ID of an Org VDC network. **Note** `.id` field of
  `vcd_network_isolated_v2`, `vcd_network_routed_v2` or `vcd_nsxt_network_dhcp` can be referenced here
```

Reply:
True. Fixed.

Reply:
Please adjust here (in resource) as well:
https://github.com/vmware/terraform-provider-vcd/pull/1039/files#diff-06e8d88888a4b61944964a8135fd659cf95cacfc1836e67f8714e71b31b9b879R60-R61

Reply:
Fixed there as well.